libbeat/beat/beat.go

```diff
@@ -84,7 +84,8 @@ type Beat struct {
 
 	Instrumentation instrumentation.Instrumentation // instrumentation holds an APM agent for capturing and reporting traces
 
-	API *api.Server // API server. This is nil unless the http endpoint is enabled.
+	API      *api.Server // API server. This is nil unless the http endpoint is enabled.
+	Registry *reload.Registry
```

Comment:
Can you please add some godoc to explain what this is and/or how it should be used to Beat authors.